ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/HiveMetaStoreAuthorizer.java

```diff
@@ -312,5 +558,9 @@ private String getCurrentUser() {
   private String getCurrentUser(HiveMetaStoreAuthorizableEvent authorizableEvent) {
     return authorizableEvent.getAuthzContext().getUGI().getShortUserName();
   }
+
+  private UserGroupInformation getUGI() throws IOException {
+      return UserGroupInformation.getCurrentUser();
```

Comment:
Wrong indentation

Reply:
all corrected. How did you spot them so quickly? I wish I have a tool to find out before I submit.

Reply:
Too many years seeing Hive code. Basically it hurts my eyes :)